src/RIGID/fix_rigid.cpp

```diff
@@ -991,12 +991,33 @@ void FixRigid::apply_langevin_thermostat()
 
       gamma1 = -1.0 / t_period / ftm2v;
       gamma2 = tsqrt * sqrt(24.0*boltz/t_period/dt/mvv2e) / ftm2v;
-      langextra[i][3] = inertia[i][0]*gamma1*omega[i][0] +
+
+      // convert omega from space frame to body frame
+
```

Comment:
Is this the same as:
MathExtra::matvec(ex_space[i],ey_space[i],ez_space[i],omega[i],wbody);
and ditto below for:
MathExtra::matvec(ex_space[i],ey_space[i],ez_space[i],tbody,&langextra[3]);
I.e. replace 3 lines with 1. ??

Reply:
@sjplimp thanks for pointing to the existing helper functions, the first is transpose_matvec(), the latter is matvec()

MathExtra::transpose_matvec(ex_space[i],ey_space[i],ez_space[i],omega[i],wbody);
MathExtra::matvec(ex_space[i],ey_space[i],ez_space[i],tbody,&langextra[i][3]);

I will make the changes as requested.